styleguide/tokens/animations.md

```diff
@@ -1,10 +1,10 @@
 # Animations
 
-All animations are 0.3s long. This is defined in the <span class="html inline">$cads-animation-speed</span> variable. However you should beware of reduced motion user preferences. These can be checked using the <span class="html inline">prefers-reduced-motion: reduce</span> media query. For defining transitions use the <span class="html inline">cads-transition-animation</span> mixin. This will automatically ignore transition animations if the user has set the reduce motion option. You can supply the prop to animate or if none is supplied <span class="html inline">all</span> is used. Example:
```

Comment:
This previously read:

> "You can supply the prop to animate or if none is supplied 'all' is used". 

This wasn't true even before my changes. Not sure this is intentional. Updated the docs to reflect the defaults are `background` and `border`